app/views/catalog/record/_mods_bibliographic.html.erb

```diff
@@ -1,4 +1,9 @@
 <% document ||= @document %>
+
+<% document.mods.send(:mods_field, document.mods.instance_variable_get(:@xml), :title).fields.reject { |x| x.label =~ /^Title/i }.each do |title| %>
+  <%= mods_record_field(title) %>
+<% end %>
```

Comment:
Is this ultimately different than using `document.mods.sub_title` (which is effectively all the titles except the first)?

If not, I'm wondering if this is something that would be good to push upstream since this code is now duplicated in both PURL and SearchWorks (which we can totally split out into a separate ticket).

Reply:
And now that I look at the code, I think ModsDisplay intended to surface `sub_title` as an accessor but maybe accidentally did not.  If it would server this purpose, I think that would be a good way to approach this instead of using `send` in a view.

Reply:
I added a PR to `ModsDisplay` to add a `subTitle` accessor. sul-dlss/mods_display#58

If that is accepted, and we cut a release, we can remove the `#send` on line 3 above and replace it with:

```erb
<% document.mods.subTitle.each do |title| %>
```